code/__DEFINES/mobs.dm

```diff
@@ -540,12 +540,15 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 //Defiler defines
 #define DEFILER_GAS_CHANNEL_TIME 0.5 SECONDS
 #define DEFILER_GAS_DELAY 1 SECONDS
-#define DEFILER_STING_CHANNEL_TIME 1.5 SECONDS
+#define DEFILER_STING_CHANNEL_TIME 0.3 SECONDS
```

Comment:
It should be fast.

Consider the Spitter scatterspit; that's 0.5 seconds, and people rightfully complain about it being too long, and you even agreed they're right to complain.

Reply:
The channel time for neuro stings is 0.5, and I think it's fair. It's still possible to do stings (even as a sentinel, a very squishy xeno).

Is the cooldown procing when you miss your sting ?

Reply:
No, but I could add a partial cooldown for whiffing. I think 0.3 is fine; it's supposed to not rely on other xenos to set up, but at the same time it can be interrupted by covering fire or quick thinking marines.